kafka/kafkatest/utils.go

```diff
@@ -86,11 +88,12 @@ func NewFakeKafkaConsumer(log logrus.FieldLogger, distri <-chan *kafkalib.Messag
 func (f *FakeKafkaConsumer) FetchMessage(ctx context.Context) (*kafkalib.Message, error) {
 	for {
 		f.msgMu.Lock()
-		if int64(len(f.messages)) > f.offset {
-			f.log.WithField("offset", f.offset).Trace("offering message")
-			msg := f.messages[f.offset]
+		if int64(len(f.messages)) > f.uncommitedOffset {
+			f.log.WithField("offset", f.uncommitedOffset).Trace("offering message")
+			msg := f.messages[f.uncommitedOffset]
 			f.msgMu.Unlock()
 
+			f.uncommitedOffset = f.uncommitedOffset + 1
```

Comment:
This seems unrelated to your PR, but do we need to bump `offset` here: https://github.com/netlify/netlify-commons/blob/a8a24e784a013eb660f38b141e33c6d7585f12c7/kafka/kafkatest/utils.go#L76 ?

Reply:
I think the intent was that the `offset` was the committed offset, so that would not be touched unless we called `CommitOffset` or `Seek`. The linked code is setting the offset for incoming messages that have not been fetched yet.